content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -353,7 +352,11 @@ In the CLI, the access modes are abbreviated to:
 * ROX - ReadOnlyMany
 * RWX - ReadWriteMany
 
-> __Important!__ A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time.
+{{< caution >}}
+
+A volume can only be mounted using one access mode at a time, even if it supports many.  For example, a GCEPersistentDisk can be mounted as ReadWriteOnce by a single node or ReadOnlyMany by many nodes, but not at the same time. Besides, it will be limited once a volume has been mounted on a single node with `ReadWriteOnce`, and pod which using this volume will be [checked](https://kubernetes.io/docs/concepts/scheduling/kube-scheduler/#filtering) whether it fit the volume mount in scheduling stage.
```

Comment:
nit: “whether it fit the volume mount in scheduling stage” doesn't seem like idiomatic English. I'm also not sure what meaning you want to convey here @Colstuwjx.

Reply:
Okay, I removed this line.